docs/DeveloperGuide.md

```diff
@@ -484,25 +489,83 @@ Extensions:
 
 **Use case: UC0X - Create a session**
 
-**Preconditions: Student profile linked to session has been created.**
+MSS:
+
+1. User enters the `add_session` command, together with the session details.
+2. TutorBuddy verifies that the student exists, and the inputs are valid.
+3. TutorBuddy creates the session.
+4. TutorBuddy displays a success message.
+
+   Use case ends.
+
+**Extensions:**
+
+* 2a. TutorBuddy detects an error in the input.
+    * 2a1. TutorBuddy displays an error message to the user.
+
+  Use case ends.
+
+* 2b. TutorBuddy detects another overlapping session that the user has in the same timeframe.
+    * 2b1. TutorBuddy prompts an error and requests for the correct data.
+
+  Use case ends.
+
+**Use case: UC0X - Create a recurring session**
 
 MSS:
 
-1. User enters the add session command, together with the session details.
-2. TutorBuddy creates the session.
-3. TutorBuddy displays a success message.
+1. User enters the `add_rec_session` command, together with the session details.
+2. TutorBuddy verifies that the student exists, and the inputs are valid.
+3. TutorBuddy creates the recurring session.
+4. TutorBuddy displays a success message.
 
    Use case ends.
 
 **Extensions:**
 
-* 1a. TutorBuddy detects an error in the entered data.
-    * 1a1. TutorBuddy prompts an error and requests for the correct data.
+* 2a. TutorBuddy detects an error in the input.
+    * 2a1. TutorBuddy displays an error message to the user.
+
+  Use case ends.
+
+* 2b. TutorBuddy detects another overlapping session that the user has in the same timeframe.
+    * 2b1. TutorBuddy prompts an error and requests for the correct data.
+
+  Use case ends.
+
+**Use case: UC0X - Delete a session**
+
+MSS:
+
+1. User enters the `delete_session` command with the appropriate inputs.
+2. TutorBuddy verifies that the student exists, and the inputs are valid.
```

Comment:
Also have to verify whether session exist maybe?

Reply:
Same as above ah, cos if we want to add the check if the student / session exist, then we need to do for all liao ah

Reply:
ok!